### PR TITLE
Add volume control to ocf-tv

### PR DESCRIPTION
**Pull request by tahabi**
Now you can do `$ ocf-tv volume (0, 150)` to change the volume of the TV from 0% to 150% of baseline. It seems to work pretty decently, occasionally asks for you for a password to tornado. `connect` is still the default. 

When it does ask for a password however, `connect` will timeout if you aren't able to type the pwd fast enough, and after that fails, the terminal begins to exhibit weird behavior - no newlines on enter, no characters shown when typing, although it stills reads the input and executes when you press enter. Need to spawn a new terminal to fix it. Can't really explain why that happens when login fails.

### CODE REVIEW

File: staff/lab/ocf-tv [hunk 1]
```diff
@@ -37,7 +38,15 @@ def wait_for_port(host, port, timeout=5):
         if spent > timeout:
             raise TimeoutError('Timed out after {} seconds.'.format(timeout))
```
Comment (chriskuehl) [inline staff/lab/ocf-tv]: pep8 says to put 2 blank lines between methods

File: staff/lab/ocf-tv [hunk 2]
```diff
@@ -50,13 +59,26 @@ def connect(args):
...
+    #subparser_connect.add_argument('command', type=str, nargs='?', choices=commands.keys(), default='connect')
+
+    subparser_volume = command_subparser.add_parser('volume', help='set the volume for a PulseAudio sink on the TV')
+    subparser_volume.add_argument('--sink', '-s', required=False, type=str, default='0')
+    subparser_volume.add_argument('amount', type=volume_in_range, default='100', help='volume in percent between 0 and 130')
+
+    if len(sys.argv) == 1:
+        args = parser.parse_args(['connect'])
+    else:
+        args = parser.parse_args() # default is sys.argv
```
Comment (chriskuehl) [inline staff/lab/ocf-tv]: pep8 says to put 2 spaces before inline comments

File: staff/lab/ocf-tv [hunk 3]
```diff
@@ -50,13 +59,26 @@ def connect(args):
...
+    
+    # args = parser.parse_args()
```
Comment (chriskuehl) [inline staff/lab/ocf-tv]: delete?

File: staff/lab/ocf-tv [hunk 4]
```diff
@@ -50,13 +59,26 @@ def connect(args):
...
     )
     parser.add_argument('-H', '--host', type=str, default='tv')
-    parser.add_argument('command', type=str, nargs='?',
-                        choices=commands.keys(), default='connect')
 
-    args = parser.parse_args()
+    command_subparser = parser.add_subparsers(dest='command', help='command to run')
+
+    subparser_connect = command_subparser.add_parser('connect', help='open a vnc instance to view the TV screen')
+    #subparser_connect.add_argument('command', type=str, nargs='?', choices=commands.keys(), default='connect')
```
Comment (chriskuehl) [inline staff/lab/ocf-tv]: delete?

File: staff/lab/ocf-tv [hunk 5]
```diff
@@ -38,25 +39,49 @@ def wait_for_port(host, port, timeout=5):
...
+
+
 def connect(args):
     port = unused_port()
     proc = Popen(('ssh', '-L', '{}:localhost:5900'.format(port), '-N', args.host))
     wait_for_port('localhost', port)
     call(('xvnc4viewer', 'localhost:{}'.format(port)))
     proc.terminate()
-    proc.wait()
+    proc.wgait()
```
Comment (chriskuehl) [inline staff/lab/ocf-tv]: durp

Comment (tahabi) [inline staff/lab/ocf-tv]: lol my finger slipped on an emacs M-g and I didn't catch it

**Pull request merged**